source/_includes/asides/docs_sitemap.html

```diff
@@ -0,0 +1,290 @@
+        <li>{% active_link /voice_control/create_wake_word/ Tutorial: Create your own wake word %}</li>
+        <li>{% active_link /voice_control/assist_daily_summary/ Tutorial: Your daily summary by Assist %}</li>
+      </ul>
+    </li>
+    <li>{% icon "mdi:account-help" %} Troubleshooting
+      <ul>
+        <li>{% active_link /voice_control/troubleshooting/ Troubleshooting Assist %}</li>
+        <li>{% active_link /voice_control/troubleshooting_the_s3_box/ Troubleshooting the ESP32-S3-BOX %}</li>
+      </ul>
+    </li>
+  </ul>
+  {% endif %}
+</li>
+
+<li>
+  {% icon "mdi:category" %} {% active_link /docs/organizing/ Organization %}
+  {% if doc == 'organizing' or include.docs_index %}
+  <ul>
+    <li>{% active_link /docs/organizing/ Grouping your assets %}</li>
+    <li>{% active_link /docs/organizing/tables Working with tables %}</li>
+    <li>{% active_link /docs/organizing/areas/ Areas %}</li>
+    <li>{% active_link /docs/organizing/floors/ Floors %}</li>
+    <li>{% active_link /docs/organizing/labels/ Labels %}</li>
+    <li>{% active_link /docs/organizing/categories/ Categories %}</li>
+    <li>{% active_link /docs/frontend/icons/ Icons %}</li>
+  </ul>
+  {% endif %}
+</li>
+
+<li>
+  {% icon "mdi:solar-power-variant" %} {% active_link /docs/energy/ Home energy management %}
+  {% if doc == 'energy' or include.docs_index %}
+  <ul>
+    <li>{% active_link /docs/energy/electricity-grid/ Electricity grid %}</li>
+    <li>{% active_link /docs/energy/solar-panels/ Solar panels %}</li>
+    <li>{% active_link /docs/energy/individual-devices/ Individual devices %}</li>
+    <li>{% active_link /docs/energy/faq/ FAQ %}</li>
+  </ul>
+  {% endif %}
+</li>
+
+<li>
+  {% icon "mdi:cogs" %} {% active_link /docs/configuration/ Advanced configuration %}
+  {% if doc == 'configuration' or include.docs_index %}
+  <ul>
+    <li>{% active_link /docs/configuration/yaml/ YAML syntax %}</li>
+    <li>{% active_link /docs/configuration/basic/ Basic information %}</li>
+    <li>{% active_link /docs/configuration/customizing-devices/ Customizing entities %}</li>
+    <li>{% active_link /docs/configuration/troubleshooting/ Troubleshooting configuration %}</li>
+    <li>{% active_link /docs/configuration/securing/ Security check points %}</li>
+    <li>{% active_link /docs/configuration/remote/ Remote access %}</li>
+    <li>{% active_link /docs/configuration/splitting_configuration/ Splitting up the configuration %}</li>
+    <li>{% active_link /docs/configuration/packages/ Packages %}</li>
+    <li>{% active_link /docs/configuration/secrets/ Storing secrets %}</li>
+    <li>{% active_link /docs/configuration/events/ Events %}</li>
+    <li>{% active_link /docs/configuration/state_object/ State objects %}</li>
+    <li>{% active_link /docs/configuration/templating/ Templating %}</li>
+    <li>{% active_link /docs/configuration/platform_options/ Entity component platform options %}</li>
+  </ul>
+  {% endif %}
+</li>
+
+<li>
```

Comment:
should common tasks be placed further up, above Advanced configuration? It covers tasks such as update and backup, as well as configuring file access and setting up an editor, which is used for advanced config.

Reply:
Sure, that makes sense. 😊 This PR is mostly about the UI so I tried to make as few editorial decisions as I could.

Reply:
Done in [baca135](https://github.com/home-assistant/home-assistant.io/pull/33601/commits/baca135471f19d298f613e1bc41b8a696c929e90)